main_app/templates/main_app/helpline_numbers.html

```diff
@@ -65,13 +65,53 @@
 
             }
         }
+<!-----------------------------------------------------------Top Button---------------------------------------------------------------------------->
```

Comment:
Remove this comment. Because of this `scroll-behavior` is getting hampered. You can check this.